omero/sysadmins/public.rst

```diff
@@ -138,3 +139,132 @@ authentication. For example a direct link to image will look as follows::
 For more details about how to create an OMERO session see
 :doc:`server-side session </developers/Server/Sessions>` or
 use the :doc:`command line interface </users/cli/sessions>` to create one.
+
+.. _hosting_data_example:
+
+Complete workflow for hosting data for a publication
+----------------------------------------------------
+
+Putting the pieces of this puzzle together, the following describes the steps
+of a complete workflow for using OMERO to host public data associated with a
+publication. It is illustrated using an example publication from the Swedlow
+lab in Dundee,
+`Schleicher et al, 2017 <http://dx.doi.org/10.1098/rsob.170099>`_ with the
+data hosted at
+`<https://omero.lifesci.dundee.ac.uk/pub/schleicher-et-al-2017>`_.
+
+Ansible playbooks can be found describing how the "nightshade" server was
```

Comment:
better to use ``how the production server in Dundee`` was ...

Reply:
I purposefully used "nightshade" because otherwise it isn't clear what playbooks to read in that repo. Otherwise, we need to rename the files.

Reply:
in that case maybe the suggested text and in brackets or i.e. "nightshade"